aptos-move/aptos-transaction-benchmarks/benches/transaction_benches.rs

```diff
@@ -18,10 +18,10 @@ fn peer_to_peer<M: Measurement + 'static>(c: &mut Criterion<M>) {
         bencher.bench(b)
     });
 
-    c.bench_function("peer_to_peer_parallel", |b| {
-        let bencher = TransactionBencher::new(any_with::<P2PTransferGen>((1_000, 1_000_000)));
-        bencher.bench_parallel(b)
-    });
+    // c.bench_function("peer_to_peer_parallel", |b| {
```

Comment:
revert?

Reply:
I actually fixed it now so it can benchmark with _parallel (without having to make any code changes each time).